### PR TITLE
Update mypy.ini to call django-stubs and fix policy errors

### PR DESCRIPTION
**Pull request by JustEmrick**
# Description
For this pull request, I wanted to tackle Issue 143 and fix all the mypy errors. As part of it, I noticed a lot of the errors were related to how mypy was reading Django so I wanted to use the [django-stubs](https://github.com/typeddjango/django-stubs) instead of ignoring those errors. 

### Pre Django-stubs
This was the amount of errors that mypy was displaying before using django-stubs:
image

### Post Django-stubs
This was how it looks after:
image

### Fixed Errors
After correcting the remaining errors
image

Closes Issue #143.

### CODE REVIEW

File: chirps/policy/views.py [hunk 1]
```diff
@@ -94,11 +94,11 @@ def clone(request, policy_id):
         )
 
     # Redirect to the edit page for the new policy
-    return redirect('policy_edit', policy_id=cloned_policy.id)
+    return redirect('policy_edit', policy_id=cloned_policy.id)   # type: ignore
```
[the remark this answers is not in the document]
Comment (JustEmrick) [reply chirps/policy/views.py]: I meant to ignore just the .`id`. Instead, I did this
id = models.AutoField(primary_key=True)

File: chirps/policy/models.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ class Policy(models.Model):
         'PolicyVersion', on_delete=models.CASCADE, related_name='current_version', null=True, blank=True
     )
 
-    def __str__(self):
+    def __str__(self: 'Policy') -> str:
```
[the remark this answers is not in the document]
Comment (JustEmrick) [reply chirps/policy/models.py:27]: For some reason it errors if I don't. It's the only one that seems to do it.

**Review by JustEmrick — COMMENTED** [review]
Quick update

**Comment by alex-nork** [comment]
Sorry for the merge conflicts. Feel free to remove my recent change to `mypy.ini` in favor of your use of the django-stubs plugin.

**Comment by JustEmrick** [comment]
Closing as I rebased and created a new branch and pull request as there were many conflicts.

**Pull request closed without merging**